### PR TITLE
asadiqbal08/Add support for the default Feature Image

### PR DESCRIPTION
**Pull request by asadiqbal08**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots

#### What are the relevant tickets?
fixes: #99 

#### What's this PR do?
Added default image if nothing is provided by content manager. 

#### How should this be manually tested?
Create a product without featured image, default one should rendered as shown below.

#### Screenshots (if appropriate)
Screen Shot 2021-08-30 at 3 12 59 PM
Screen Shot 2021-08-31 at 12 45 32 PM

screencapture-localhost-8013-2021-08-31-12_45_51

Screen Shot 2021-08-31 at 1 01 49 PM

### CODE REVIEW

**Review by arslanashraf7 — CHANGES_REQUESTED** [review]
I have added a comment which seems reasonable to implement at this point but if there are any set backs to that you can let me know.

Also, could you add desktop and CMS/Wagtail screenshots in the PR description as well. for Peter to review.

File: cms/models.py
Comment (asadiqbal08) [reply cms/models.py:116]: @arslanashraf7 I looked over it, I guess, in Django we need to either define a template tag or filter if we need to achieve this as you suggested above because we need to pass the product back on the Property to calculate the result.

Comment (arslanashraf7) [reply cms/models.py:116]: I think we already have a tag for images, How feasible do you think it would be to update to support default images? https://github.com/mitodl/mitxonline/blob/5ad9aa49161765828a788e7bf926defdf5e1d8a8/cms/api.py#L102

**Review by arslanashraf7 — CHANGES_REQUESTED** [review]
I requested a test and a small change. After that this should be good to merge.

**Review by arslanashraf7 — APPROVED** [review]
Looks good 👍 

Please get the Screenshots approved by @pdpinch since it adds a placeholder and the design needs approval.

**Comment by asadiqbal08** [comment]
@pdpinch are we good to merge it ?

**Review by pdpinch — APPROVED** [review]
👍

**Pull request merged**
